stacker/actions/build.py

```diff
@@ -231,7 +231,14 @@ def run(self, outline=False, tail=False, dump=False, *args, **kwargs):
                 plan.dump(directory=dump, context=self.context)
 
     def cleanup(self):
-        logger.debug('Cleaning up')
+        """Cleans up any resources that do not need to exist after stacker
+        has finished building all the stacks.
+
+        This is specifically used when deleting the sqs queues used for
+        polling events.
+
+        """
+        logger.debug('Cleaning up sqs queues')
```

Comment:
I'd leave this a bit more generic - since this isn't actually what cleans up the queues, but the provider call itself (so putting a log message in there like this makes more sense)